umbraco/Umbraco-CMS
# Miniprofiler for v13

### Description
- Configured miniprofiler to show reports in [/umbraco/profiler/results-index](https://localhost:44339/umbraco/profiler/results-index)
- Configured to ignore endpoints we cannot do anything about, like swagger and the miniprofiler ones.
- Configured the profilers to have the url path as name

### Known limitation
- Currently we cannot protect the endpoint. A FIXME is added

### Tests
- Ensure debug is enabled in config
 - Verify results is shown in [/umbraco/profiler/results-index](https://localhost:44339/umbraco/profiler/results-index)
 - Go try out endpoints from management API or web requests and verify those are shown
 
  
![image](https://user-images.githubusercontent.com/1561480/219336255-a5bdbafe-92cf-449b-a66d-5e2d891ebb61.png)

 
- Ensure debug is disabled in config
  - Verify the report is empty [/umbraco/profiler/results-index](https://localhost:44339/umbraco/profiler/results-index)
  - Go try out endpoints from management API or web requests and verify those are NOT shown
  
![image](https://user-images.githubusercontent.com/1561480/219335603-3706f2da-0354-4e9c-93bf-ae30c8edf4b0.png)
